import-scripts/import-genie-data.sh

```diff
@@ -29,7 +29,12 @@ FLOCK_FILEPATH="/data/portal-cron/cron-lock/import-genie-data.lock"
     fi
     JAVA_IMPORTER_ARGS="$JAVA_PROXY_ARGS $java_debug_args $JAVA_SSL_ARGS -Dspring.profiles.active=dbcp -Djava.io.tmpdir=$tmp -ea -cp $IMPORTER_JAR_FILENAME org.mskcc.cbio.importer.Admin"
     genie_portal_notification_file=$(mktemp $tmp/genie-portal-update-notification.$now.XXXXXX)
-    ONCOTREE_VERSION_TO_USE=oncotree_2018_06_01
+    IMPORTING_GENIE_RELEASE_BEFORE_9_0=0
+    if [ $IMPORTING_GENIE_RELEASE_BEFORE_9_0 != "0" ] ; then
```

Comment:
That is true Manda ... for now we would need to manually edit the import script and "throw" this switch before import. This new code is just a (temporary) convenience for us so that we can make a one character change in the input script (changing the 0 to a 1) and the correct old oncotree version would be used without us having to look up which version used to be the standard version.

We do want a more permanent fix to this ... and @ritikakundra is working with Tom to begin supplying an indication of which version of onctoree is appropriate in the genie release data itself (either in the meta_study.txt file or as a separate file). Once that is established we can write some code to use the version identifier present in the genie data.

Reply:
If you think it would preferable, we can change this script so that when it is called it has a required argument to specify the oncotree version to use ... and we can have a comment note in the crontab which tells us the old oncotree version to use for pre-9.0 genie releases. Would that be better?

Reply:
merging this PR because it has been deployed and in production without problems